tests/data_generators.py

```diff
@@ -69,8 +69,11 @@ def create_member(
         name: str = "test member name",
         account: Optional[Account] = None,
         password: str = "password",
-        registered_on: datetime = None,
+        registered_on: Optional[datetime] = None,
+        is_confirmed: Optional[bool] = None,
     ) -> Member:
+        if is_confirmed is None:
```

Comment:
`is_confirmed` is not used in this method. are you planning to do something with this later?